examples/packet-production/cluster.lokocfg

```diff
@@ -57,6 +57,10 @@ cluster "packet" {
   cluster_name     = var.cluster_name
   controller_count = var.controllers_count
 
+  //os_channel       = "stable"
+  //os_version       = "current"
```

Comment:
I think `os_version` doesn't work on Packet. We used preinstalled Flatcar images on amd64, so we can't choose the version, we just run "whatever the preinstalled version is".

On ARM we use IPXE, IIRC, and we can choose that, though

Reply:
@rata is right. On Packet, specifying `os_version` only will cause configuration error, as recently added in https://github.com/kinvolk/lokomotive/pull/540.

Reply:
I think for Packet, we could just set commented out `os_channel` value then.

Reply:
I can changed os_version to "" since that's the only value accepted without `ipxe_script_url`. It is commented.